mftutor/reg/views.py

```diff
@@ -38,6 +41,23 @@ class BurStartView(TemplateView):
 
 # =============================================================================
 
+class KrydslisteCreateForm(forms.Form):
+    value = forms.CharField(label='input')
+
+class KrydslisteView(FormView):
+    template_name = 'reg/krydsliste.html'
+    form_class = KrydslisteCreateForm
+
+    def form_valid(self, form):
+
+        pdf = PDF(orientation='P', unit='mm', format='A3')
+        pdf.create(form.cleaned_data['value'].splitlines())
+        pdf.output('test.pdf', 'F')
+        openPDF = open('test.pdf', 'rb').read()
+        os.remove('test.pdf')
+        return HttpResponse(openPDF, content_type='application/pdf')
```

Comment:
Vil dette virke i stedet? Så slipper man for at lave en fil på disken.

```suggestion
        res = pdf.output(dest='S')
        return HttpResponse(res, content_type='application/pdf')
```

Reply:
Nej det virker ikke, det outputter bare en blank PDF

Reply:
Hmm okay. Jeg synes som minimum man skal bruge et mere robust midlertidigt filnavn end "test.pdf". Hvad med:

```suggestion
        with tempfile.NamedTemporaryFile() as f:
            res = pdf.output(f.name, 'F')
            return HttpResponse(f.read(), content_type='application/pdf')
```

Reply:
Kan du prøve at uploade et eksempel på den blanke PDF den laver med den første løsning jeg foreslog? Jeg synes det lyder som en bug i fpdf at den laver en blank PDF...